src/map/cashshop.c

```diff
@@ -162,7 +121,7 @@ static int cashshop_read_db_sql( void ){
 				}
 			}
 
-			if( !cashshop_parse_dbrow( str, cash_db_name[fi], lines ) )
+			if( !cashshop_parse_dbrow( str, 3, lines ) )
```

Comment:
well, then add the info on fail here, tells the SQL table name, since in previous function it parses file or sql name.
something like
````
if( !cashshop_parse_dbrow( str, 3, lines ) ) {
    ShowError("cashshop_read_db_sql: Cannot process table '%s' at line '%d'.\n", what_is_sql_table_var, lines);
    continue;
}
````